app.js

```diff
@@ -40,6 +40,7 @@ app.use(express.urlencoded({ extended: false }))
 app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 app.use((req, res, next) => {
+  res.append('X-Frame-Options', 'SAMEORIGIN')
```

Comment:
can we handle this in middleware

Reply:
@vaibhavdaren 
I am confused 😅. Isn't that an application level middleware function 🤔. Are you pointing to something else?